mercury/forms.py

```diff
@@ -1,22 +1,35 @@
 from django import forms
-from mercury.models import SimulatedData
+from mercury.models import (
+    TemperatureSensor,
+    AccelerationSensor,
+    WheelSpeedSensor,
+    SuspensionSensor,
+    FuelLevelSensor,
+)
 
-# for our slider
-from django.forms.widgets import NumberInput
 
-
-class RangeInput(NumberInput):
-    input_type = "range"
-
-
-class SimulatorForm(forms.ModelForm):
+class TemperatureForm(forms.ModelForm):
     class Meta:
-        model = SimulatedData
+        model = TemperatureSensor
         fields = "__all__"
         widgets = {
             "temperature": forms.NumberInput(
                 attrs={"id": "post-temperature", "required": True}
             ),
+            "created_at": forms.DateTimeInput(
+                attrs={"id": "post-created-at", "required": True}
+            ),
+        }
+
+
+class AccelerationForm(forms.ModelForm):
+    class Meta:
+        model = AccelerationSensor
+        fields = "__all__"
+        widgets = {
+            "created_at_accel": forms.DateTimeInput(
+                attrs={"id": "post-created-at_accel", "required": True}
```

Comment:
You can leave the `id` of this as-is (currently post-created-at_accel) in that `attrs` if that's how you want to identify it in the JavaScript code, but actual field in the data model needs to be simply `created_at`. This same comment applies to the other same situations in this model.